packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<SettingsView>
+		<div :class="$style.container">
+			<div>
+				<n8n-heading size="2xlarge">{{ $locale.baseText('settings.communityNodes') }}</n8n-heading>
+			</div>
+			<div :class="$style.actionBoxContainer">
+				<n8n-action-box
+					:heading="$locale.baseText('settings.communityNodes.empty.title')"
+					:description="getEmptyStateDescription"
+					:buttonText="$locale.baseText('settings.communityNodes.empty.installPackageLabel')"
+					:calloutText="actionBoxConfig.calloutText"
+					:calloutTheme="actionBoxConfig.calloutTheme"
+					:hideButton="actionBoxConfig.hideButton"
+					@click="openNPMPage"
+				/>
+			</div>
+		</div>
+	</SettingsView>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import SettingsView from './SettingsView.vue';
+export default Vue.extend({
+	name: 'SettingsCommunityNodesView',
+	components: {
+		SettingsView,
+	},
+	async mounted() {
+		await this.$store.dispatch('communityNodes/fetchAvailableCommunityPackageCount');
```

Comment:
should be wrapped in try/catch... in case of error, no need to do anything since it's a minor label